src/MessageFactory/DiactorosMessageFactory.php

```diff
@@ -11,6 +11,8 @@
  * Creates Diactoros messages.
  *
  * @author GeLo <geloen.eric@gmail.com>
+ *
+ * @deprecated This will be removed in 2.0. Consider using the official Diactoros PSR-17 factory
```

Comment:
We may want to include the package name together with the version. This way, it will be more obvious which package deprecated the interface (the Symfony Debug component turns the `@deprecated` annotation into `E_USER_DEPRECATED` errors IIRC).